app.py

```diff
@@ -62,25 +61,49 @@ def flights():
     return "/flights?budget=xxx&seg=x&origin1=xxx&dest1=xxx&date1=xxxx-xx-xx"
 
 
-def get_flights(raw_input):
-    qpx.get(raw_input)
 
+def get_flights(raw_input):
+    file = qpx.get(raw_input)
     logging.warning(qpx.data)
-    print(qpx.data, file=open('latest_query.log', 'w+'))
+    # re-formated JSON returned data
+    return_data = dict()
+    # only return top 50 flight data 
+    for x in range(50):
+        option = dict()
+        option['price%d' % x] = qpx.data['trips']['tripOption'][x]['saleTotal']
+        segment = dict()
+        for y in range(len(qpx.data['trips']['tripOption'][x]['slice'])):
+            stops = dict()
+            for z in range(len(qpx.data['trips']['tripOption'][x]['slice'][y]['segment'])):
+                segs = qpx.data['trips']['tripOption'][x]['slice'][y]['segment'][z]
+                stop = dict()
+                stop['carrier'] = segs['flight']['carrier']
+                stop['flight_number'] = segs['flight']['number']
+                stop['arrivalTime']= segs['leg'][0]['arrivalTime']
+                stop['departureTime'] = segs['leg'][0]['departureTime']
+                stop['origin'] = segs['leg'][0]['origin']
+                stop['destination'] = segs['leg'][0]['destination']
+                stop['destinationTerminal'] = segs['leg'][0]['destinationTerminal']
+                stops["%d" % z] = stop
+            segment['%d' % y] = stops
+        option['segment%d' % x] = segment
+        return_data['option%d' % x] = option
+    print(return_data, file=open('top_query.log', 'w+'))
```

Comment:
I think we could upload an example query return to Github for the front-end team to better understand the structure of the output.